src/jit/gentree.cpp

```diff
@@ -5084,6 +5084,8 @@ bool GenTree::TryGetUse(GenTree* def, GenTree*** use)
         case GT_NOP:
         case GT_RETURN:
         case GT_RETFILT:
+        case GT_BSWAP:
+        case GT_BSWAP16:
```

Comment:
It looks like you missed a similar change in compiler.hpp `GenTree::VisitOperands`.

Also, you may want to add these to `Compiler::optIsCSEcandidate` in optcse.cpp as well.

Hopefully that's all of it.